tests/ui/indexing/index_message.stderr

```diff
@@ -1,8 +1,8 @@
 error[E0608]: cannot index into a value of type `({integer},)`
-  --> $DIR/index_message.rs:3:13
+  --> $DIR/index_message.rs:3:14
    |
 LL |     let _ = z[0];
-   |             ^^^^ help: to access tuple elements, use: `z.0`
+   |              ^^^ help: to access tuple elements, use: `.0`
```

Comment:
maybe this got a little worse by not saying `z.0` in the message anymore but mentioning the LHS could quickly get out of hand with larger lhs exprs so I don't think this is a problem.

Reply:
Ideally it would include the LHS if it is short, and if the LHS is longer (e.g. the criterion could be whether it's multiline or not), it would not include the LHS.